designer/client/cypress/e2e/fragment.cy.ts

```diff
@@ -238,16 +238,16 @@ describe("Fragment", () => {
 
         cy.get("[model-id=sendSms]").should("be.visible").trigger("dblclick");
         cy.intercept("POST", "/api/nodes/*/validation", (request) => {
-            if (request.body.nodeData.ref?.parameters[0]?.expression.expression == "#fragmentResult.") {
```

Comment:
i have a feeling, that this test was about verification, whether suggestion works after the typing `.` value